.github/workflows/pr-checks.yaml

```diff
@@ -150,24 +150,12 @@ jobs:
 
       - name: Run unittests for policies
         run: |
-          /usr/bin/opa test policies unittests unittests/test_data -c > /tmp/policy_unittest_result.json
+         TEST_FILES="./test ./policies ./unittests ./unittests/test_data"
```

Comment:
`./unittests/test_data` data doesn't contain any tests, they should be probably omitted here

Reply:
I think `./unittests/test_data` is the tested data, so we need load it in `opa test` command.

Reply:
ah sorry, you are right, it's not related to coverage itself